### ISSUE TITLE
Line -1 when included template not found

### ISSUE THREAD

**Issue by spaze** [efabrica-team/phpstan-latte#252]
I have a template file with `{include "file that does not exist"}`, for example like this (notice the extra `}` after `.latte` and before `"`):

```html
<div class="column-sidebar">
	{include "../Post/common/sidebar.latte}"}
</div>
```

The error correctly says the file doesn't exist but the line shows `-1`:

```
 ------ ----------------------------------------------------------------------- 
  Line   Www/Presenters/templates/Talks/../Post/common/sidebar.latte} rendered  
         from MichalSpacekCz\Www\Presenters\TalksPresenter::talk included in    
         Www/Presenters/templates/Talks/talk.latte                              
 ------ ----------------------------------------------------------------------- 
  -1     Template file                                                          
         "/home/runner/work/michalspacek.cz/michalspacek.cz/site/app/Www/Prese  
         nters/templates/Talks/../Post/common/sidebar.latte}" doesn't exist.    
 ------ ----------------------------------------------------------------------- 
```

Can the line number be changed to where the wrong include is in the template?

**Comment by MartinMystikJonas** [comment]
I have to check wha it is not reported correctly. It should work I think.

**Comment by lulco** [comment]
It's because it is just collected via collector and then we are trying to analyse it. We have to create node visitor

**Comment by MartinMystikJonas** [comment]
ColleczedTemplateRender contains info about file and line where it was collected

**Comment by spaze** [comment]
Looking at the message, the error seems fine, technically. Because it says it happened in the non-existent file so any other line number would be strange.

But it would be cool if the error happened in the file that includes the non-existent file, talk.latte in this case. But even currently it's useful, I'd miss that typo.

**Comment by MartinMystikJonas** [comment]
It should be reported at the place where include is located in calling template. We must have some bug there.

**Comment by MartinMystikJonas** [comment]
Oh yes we report correct location if included template path expression is not resolved or empty. But when it is vallid (even non-existent) path we find out only when we try to compile it. Should be enough to check for file existence at same olace where we check it it is not empty/unresolvable.

**Comment by MartinMystikJonas** [comment]
Solved by #255
